episodes/22-external.md

```diff
@@ -26,8 +20,40 @@ exercises: 25
+  - **contact information** - include email addresses or mailing lists or other details on how to get in touch with the lesson maintainers.
+  - **contributing** - this is an opportunity to list what kinds of contribution are sought and how to get involved in lesson development for new contributors. You can provide more details in a separate `CONTRIBUTING.md` file within the repository’s root directory and link to it from `README`.
+  - **credits/acknowledgements** - make sure to credit those who have helped in the lesson's development or inspired it, and/or any resources/templates that you have reused. You can also link to a separate `AUTHORS.md` file within the repository’s root directory to list all the people who contributed to the lesson content (if this list starts to become too large to include in the `README` itself).
+  - **citation** - it’s a very good idea to specify a reference to an appropriate academic publication so others can cite the use of the lesson in their own publications and media. A common practice is to include the citation information in a separate `CITATION.md` file within the repository’s root directory and link to it from `README`.
+  - **licence** - a short description of and a link to the lessons’s licence typically contained in a separate `LICENSE.md` file within the repository’s root directory.
+
+### Instructions for contributors - `CONTRIBUTING.md`
 
+`CONTRIBUTING.md` is a file within the repository's root directory where you should provide detailed description of the ways people can send their contributions, what kinds of contribution will be credited and in what ways. For example, at what point would someone be listed as an author and how you will credit contributions that are not recorded in the commit history of the project. The latter is particularly important for newcomers who may not be proficient with the use of GitHub's features for collaborative work, such as issues, mentions, pull requests and code review, but could still provide valuable input and contributions.
+
+### Issue and pull request templates
+
+Consider setting up issue and pull request templates to help newcomers who may not have much experience working on collaborative projects in GitHub. Such templates will prompt them to fill in answers to pre-set questions, which will help raise issues or submit pull requests in a way that’s clear, helpful and provides enough information for maintainers to act upon (without going back and forth to extract it). GitHub provides a range of default templates, but you can also [write your own](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository).
+
+### Other documentation
+
+Once have set up the basic documentation about your lesson, you may consider adding the following useful information to your documentation too:
+
+- set-up guides for installing and rendering the lesson locally
```

Comment:
we should add a link here to instructions for this. probably https://carpentries.github.io/sandpaper-docs/, right @zkamvar?

Reply:
Correct!

Reply:
```suggestion
- set-up guides for installing and rendering the lesson locally. In most cases, a link to [The Carpentries Workbench documentation](https://carpentries.github.io/sandpaper-docs/) will be sufficient.
```